zenoh/src/api/session.rs

```diff
@@ -1555,10 +1555,7 @@ impl SessionInner {
         let mut state = zwrite!(self.state);
         tracing::trace!("declare_liveliness({:?})", key_expr);
         let id = self.runtime.next_id();
-        let tok_state = Arc::new(LivelinessTokenState {
-            id,
-            key_expr: key_expr.clone().into_owned(),
-        });
+        let tok_state = Arc::new(LivelinessTokenState { id });
 
         state.tokens.insert(tok_state.id, tok_state.clone());
```

Comment:
Now `state.tokens` is essentially a map form token ids to token ids; it ought to simply be a set of token ids.

Reply:
Actually, I'm not even sure we need an set.

Reply:
Well, it in fact became completely useless. I completely removed it.